packages/ui-reactive/src/Unlocking.tsx

```diff
@@ -13,7 +13,6 @@ import { TxButton } from '@polkadot/ui-app';
 import { withCalls } from '@polkadot/ui-api';
 
 type Props = BareProps & CallProps & I18nProps & {
-  balances_freeBalance?: BN,
```

Comment:
This wasn't used, my bad I guess since I've mainly touched this component.

Reply:
It happens. I've done that a couple of times as well.